test/algorithms/simulation/sidb/random_sidb_layout_generator.cpp

```diff
@@ -402,13 +405,13 @@ TEST_CASE("Random offset::ucoord_t layout generation", "[generate-random-sidb-la
     }
 }
 
-TEST_CASE("Random siqad::coord_t layout generation", "[generate_random_sidb_layout]")
+TEST_CASE("Random siqad::coord_t layout generation", "[generate-random-sidb-layout]")
 {
     SECTION("empty parameters")
     {
-        const generate_random_sidb_layout_params<sidb_cell_clk_lyt> params{};
+        const generate_random_sidb_layout_params<sidb_cell_clk_lyt_siqad> params{};
 
-        const auto lyt = generate_random_sidb_layout<sidb_cell_clk_lyt>(sidb_cell_clk_lyt{}, params);
+        const auto lyt = generate_random_sidb_layout<sidb_cell_clk_lyt_siqad>(sidb_cell_clk_lyt_siqad{}, params);
```

Comment:
```suggestion
        const auto lyt = generate_random_sidb_layout(sidb_cell_clk_lyt_siqad{}, params);
```
The compiler should be able to infer the type `Lyt` automatically, no? Please check for this one and all the others in this file.